torch/fx/experimental/symbolic_shapes.py

```diff
@@ -2394,6 +2393,11 @@ def _recording(self):
         finally:
             self.is_recording = False
 
+    # Unlike set_replacement, this records a shapeenv event
```

Comment:
Remind me what are the implications of this other than clearing the caches?

Reply:
If we don't record events, then translation validation won't replay this this event when doing bisection. Because rename unbacked to changes the state of ShapeEnv, it's important to track these.